src/cli.ts

```diff
@@ -69,15 +69,13 @@ import { rmSync } from 'fs'
         const oldErrorsCount = getTotalErrorsCount(oldErrors)
 
         const newErrorsCountMessage =
-          newErrorsCount === 0
-            ? '0 errors found'
-            : `${newErrorsCount} error${newErrorsCount > 1 ? 's' : ''} found`
+          `${newErrorsCount} error${newErrorsCount == 1 ? '' : 's'} found`
```

Comment:
Nice. Haha, I also was working on this and made almost the same change. 😅 

I think it might be nice to have it say "X _new_ errors found" to make it a little more clear:
```suggestion
          `${newErrorsCount} new error${newErrorsCount == 1 ? '' : 's'} found`
```